[17.0][IMP] mrp_mass_production_order: Mass production with one product consumed

peluko00 opened the pull request:
This pr introduces the posibility to do the mass production orders with one product consumed for each line.

cc https://github.com/APSL 4231
@miquelalzanillas @javierobcn @mpascuall @BernatObrador @ppyczko @lbarry-apsl please review

Commit:
[IMP] mrp_mass_production_order: Mass production with one product consumed

quentinDupont reviewed (commented):
Hello !

I'm not sure to understand.
It is already possible to launch productions with products without BoMs right ?

peluko00 commented:
> Hello !
> 
> I'm not sure to understand. It is already possible to launch productions with products without BoMs right ?

Hi! Yes, it's possible

quentinDupont commented:
> > Hello !
> > I'm not sure to understand. It is already possible to launch productions with products without BoMs right ?
> 
> Hi! Yes, it's possible

So what's the value of your addings sorry i don't understand nor undesrtand it on runboat ? :slightly_smiling_face: can you explain it

peluko00 commented:
> > > Hello !
> > > I'm not sure to understand. It is already possible to launch productions with products without BoMs right ?
> > 
> > 
> > Hi! Yes, it's possible
> 
> So what's the value of your addings sorry i don't understand nor undesrtand it on runboat ? 🙂 can you explain it

It's for mass production orders to select the product to consumed

peluko00 commented:
Can you review again @quentinDupont please?

quentinDupont commented:
OK so i get it now ! The UX is not very obvious but i think I lack context : do you have example of process where it's used ?

It's weird because you can choose a product_consumed_id which is not on any BoMs of product_id.

peluko00 commented:
> OK so i get it now ! The UX is not very obvious but i think I lack context : do you have example of process where it's used ?
> 
> It's weird because you can choose a product_consumed_id which is not on any BoMs of product_id.

The workflow is if you want to use BoMs you will select with_bom option and the boms will compute for every line that you have otherwise you only need to put in the line the product that you want to produce and the product that consumes that.

quentinDupont commented:
> > OK so i get it now ! The UX is not very obvious but i think I lack context : do you have example of process where it's used ?
> > It's weird because you can choose a product_consumed_id which is not on any BoMs of product_id.
> 
> The workflow is if you want to use BoMs you will select with_bom option and the boms will compute for every line that you have otherwise you only need to put in the line the product that you want to produce and the product that consumes that.

In which field of activity it is used ? Can you add a description process on readme maybe ?

Also can you be more explicit on field name, for example 
1 : Manufactured Product
2 : Consumed Product

![image](https://github.com/user-attachments/assets/85bb065b-df8f-4522-8f1a-69029d2b9902)

peluko00 commented:
> > > OK so i get it now ! The UX is not very obvious but i think I lack context : do you have example of process where it's used ?
> > > It's weird because you can choose a product_consumed_id which is not on any BoMs of product_id.
> > 
> > 
> > The workflow is if you want to use BoMs you will select with_bom option and the boms will compute for every line that you have otherwise you only need to put in the line the product that you want to produce and the product that consumes that.
> 
> In which field of activity it is used ? Can you add a description process on readme maybe ?
> 
> Also can you be more explicit on field name, for example 1 : Manufactured Product 2 : Consumed Product
> 
> ![image](https://private-user-images.githubusercontent.com/9005817/423925354-85bb065b-df8f-4522-8f1a-69029d2b9902.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIzNjU4ODYsIm5iZiI6MTc0MjM2NTU4NiwicGF0aCI6Ii85MDA1ODE3LzQyMzkyNTM1NC04NWJiMDY1Yi1kZjhmLTQ1MjItOGYxYS02OTAyOWQyYjk5MDIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTlUMDYyNjI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTYxYjA4YjA3MDJkYzA4NDQ2NjEwZDg3YjE2YjI0NDc4ZGU5Y2FkZTdiMThlOWZmY2E4YmZiOTY0M2FjMWQyMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.mM6RlYUOHPocELUGH4x_n-xpANoNNJPkN-liK-1_fns)

I changed the fields description and i added more info to the README.

quentinDupont reviewed (approved):
Thanks for all the changes !!

peluko00 commented:
/ocabot merge minor
